docs/troubleshooting/untrusted-localhost-certificate.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Troubleshoot untrusted localhost certificate in .NET Aspire
+description: Explore strategies for troubleshooting issues when working with untrusted localhost certificates in .NET Aspire.
+ms.date: 05/14/2024
+---
+
+# Troubleshoot untrusted localhost certificate in .NET Aspire
+
+This article provides guidance on how to troubleshoot issues that you might encounter when working with untrusted localhost certificates in .NET Aspire.
+
+## Symptoms
+
+Several .NET Aspire templates include ASP.NET Core projects that are configured to use HTTPS by default. If this is the first time you're running the project, ASP.NET Core prompts you to install a localhost certificate. There are situations in which you trust/install the development certificate, but you don't close all your browser windows.
```

Comment:
as noted below, the other situation is what you move too fast or are confused and decline the dev cert. that's when you need the commands

cc @DamianEdwards to confirm this command and the one below.

Reply:
VS is the one that prompts and it's the trust operation that requires prompting. The installation of the certificate is done automatically as part of the .NET "first run" experience. When running the application, ASP.NET Core will print a warning log (which should be visible in the console) saying that Kestrel was started using the dev cert but that it wasn't trusted. The command to trust is `dotnet dev-certs https --trust`

Reply:
Perfect, thank you.